File: Week 10/Assignment 10/app/src/main/java/com/example/mod10/MainActivity.java
package com.example.mod10;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;
import android.widget.LinearLayout;
import android.widget.TextView;
import com.google.android.material.textfield.TextInputEditText;
import com.google.android.material.textfield.TextInputLayout;
import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {
    ArrayList<TextView> listItems = new ArrayList<>();
    LinearLayout parentLayout;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        parentLayout = findViewById(R.id.main);
        setContentView(R.layout.activity_main);
        TextInputEditText textInput = findViewById(R.id.UserInput);
        TextView label = findViewById(R.id.textView);
        label.setText("no Items Added");
        Button button = findViewById(R.id.button);
        button.setText("Add Note");
        button.setOnClickListener(v -> {
            addItem(textInput.getText().toString());
            textInput.getText().clear();
        });
    }
    void addItem(String text){
        TextInputLayout hint = findViewById(R.id.UserInputLayout);
        TextView layout = findViewById(R.id.textView);
        if (text.equals("")){
            hint.setHint("You need to Type Something");
            return;
        }
        hint.setHint("");
        TextView newItem = new TextView(this);
        newItem.setText(text);
        listItems.add(newItem);
        parentLayout = findViewById(R.id.main);
        for (TextView tv : listItems){
            parentLayout.removeView(tv);
            parentLayout.addView(tv);
        }
        layout.setText(listItems.size() + " Items Added");
    }
}

File: Final/Final/app/src/main/java/com/example/android/finalproject/MainActivity.kt
package com.example.android.finalproject

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private val SPLASH_DELAY: Long = 5000 // Delay in milliseconds (5 seconds)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Handler().postDelayed({
            val intent = Intent(this@MainActivity, MenuActivity::class.java)
            startActivity(intent)
            finish()
        }, SPLASH_DELAY)
    }
}
